docs-src/docs/guides/walkthrough.md

```diff
@@ -602,11 +649,182 @@ However, if you intend to continue through [Deployment 3](#deployment-3-guestboo
 
 ## 2b: Deploy with Terraform
 
+As before, we can complete this deployment using Terraform and the Layer0 provider instead of the Layer0 CLI. As before, we will assume that you've cloned the [layer0-examples](https://github.com/quintilesims/layer0-examples) repo and are working in the `iterative-walkthrough/deployment-2/` directory.
+
+We'll use these files to manage our deployment with Terraform:
+
+|Filename|Purpose|
+|---|---|
+|`Guestbook.Dockerrun.aws.json`|Template for running the Guestbook application|
+|`layer0.tf`|Provisions resources; populates variables in template files|
+|`Redis.Dockerrun.aws.json`|Template for running the Redis application|
+|`terraform.tfstate`|Tracks status of deployment _(created and managed by Terraform)_|
+|`terraform.tfvars`|Variables specific to the environment and application(s)|
+
+!!! Note "Note: `terraform.tfstate`"
+    You can follow the steps in this section with no `terraform.tfstate` file, and Terraform will create it and manage it as it did before. However, if you've opted **not** to destroy the resources created in [Deployment 1b](#1b-deploy-with-terraform), you can copy your previous `terraform.tfstate` file to your current working directory and Terraform will update and delete existing resources as needed in addition to creating new ones.
```

Comment:
Would the earlier recommendation by @zpatrick to remove this in favor of instructing the user to opt to just execute `terraform destroy` between examples apply here too?